pkg/api/dataproxy.go

```diff
@@ -115,12 +118,28 @@ func ProxyDataSourceRequest(c *middleware.Context) {
 		}
 	}
 
+	outputToAuditLog(ds.Type, c)
```

Comment:
I think the function call should happen at line 129 instead. 

If the TLS cert configuration is invalid we should not write an audit log since grafana wont send any request.